queryable/parquet_queryable.go

```diff
@@ -152,10 +175,27 @@ func (p parquetQuerier) LabelValues(ctx context.Context, name string, hints *pro
 		return nil, nil, err
 	}
 
-	return util.MergeUnsortedSlices(int(limit), resNameValues...), nil, nil
+	result = util.MergeUnsortedSlices(int(limit), resNameValues...)
+	span.SetAttributes(attribute.Int("result_count", len(result)))
+	return result, nil, nil
 }
 
-func (p parquetQuerier) LabelNames(ctx context.Context, hints *prom_storage.LabelHints, matchers ...*labels.Matcher) ([]string, annotations.Annotations, error) {
+func (p parquetQuerier) LabelNames(ctx context.Context, hints *prom_storage.LabelHints, matchers ...*labels.Matcher) (result []string, annotations annotations.Annotations, err error) {
+	ctx, span := otel.Tracer("parquet-common").Start(ctx, "parquetQuerier.LabelNames")
```

Comment:
The alternative to doing the `otel.Tracer("parquet-common")` bit every time is to have an internal pkg where we do it once and then call that every time. I have no strong opinion on this.